client/signup/style.scss

```diff
@@ -307,14 +307,14 @@ body.is-section-signup .layout:not(.dops):not(.is-wccom-oauth-flow) {
 
 	.signup__step.is-user,
 	.signup__step.is-user-social {
-		button.signup-form__submit[disabled],
-		button.signup-form__submit:disabled,
-		button.signup-form__submit.disabled {
+		.button.signup-form__submit[disabled],
+		.button.signup-form__submit:disabled,
+		.button.signup-form__submit.disabled {
 			background-color: var(--studio-gray-20);
 			color: var(--color-surface);
 		}
 
-		button.signup-form__submit {
+		.button.signup-form__submit {
```

Comment:
Stop these styles bleeding into core buttons. There is still a `.signup-form__submit` in https://github.com/Automattic/wp-calypso/blob/3ab6adb2ac69b39cb94b52ad1c55b5ba3f12d7ef/client/my-sites/invites/invite-accept-logged-out/index.jsx#L153